what's new.md

```diff
@@ -261,6 +261,7 @@
 - Add `NativeXRPlugin` and `NativeXRFrame` to improve XR performance on BabylonNative ([rgerd](https://github.com/rgerd))
 - Reset XR Camera's orientation when entering an AR session for consistent experience ([RaananW](https://github.com/RaananW))
 - Enable multiview rendering to `XRProjectionLayer` with texture type "texture-array" ([#10767](https://github.com/BabylonJS/Babylon.js/issues/10767)) ([rgerd](https://github.com/rgerd))
+- Support enable / disable switch for WebXR specutator camera and be able to assign refresh rate in `enableSpectatorMode` ([daoshengmu](https://github.com/daoshengmu))
```

Comment:
No need to update the what's new. This is done automatically in the changelog update once a week.

Reply:
Resolved by a645095ed9f74295a06a0c1a81a38f414f11f216